internal/pkg/manager/workloadannotator/workloadannotator.go

```diff
@@ -88,19 +90,36 @@ func (r *PodReconciler) Reconcile(_ context.Context, req reconcile.Request) (rec
 		logger.Error(err, "could not get pod")
 		return reconcile.Result{}, errors.Wrap(err, "looking up pod in pod reconciler")
 	}
-	if kerrors.IsNotFound(err) { // this is a pod deletion, so update all seccomp profiles that were using it
+	if kerrors.IsNotFound(err) { // this is a pod deletion, so update all seccomp/selinux profiles that were using it
 		seccompProfiles := &seccompprofileapi.SeccompProfileList{}
-		if err = r.client.List(ctx, seccompProfiles, client.MatchingFields{linkedPodsKey: podID}); err != nil {
-			return reconcile.Result{}, errors.Wrap(err, "listing SeccompProfiles for deleted pod")
+		selinuxProfiles := &selinuxprofileapi.SelinuxProfileList{}
+
+		errSeccomp := r.client.List(ctx, seccompProfiles, client.MatchingFields{linkedPodsKey: podID})
+		errSelinux := r.client.List(ctx, selinuxProfiles, client.MatchingFields{linkedPodsKey: podID})
+
+		if errSeccomp != nil && errSelinux != nil {
```

Comment:
this was intended, we don't want to continue if we can not list either seccomp profiles or selinux profiles

Reply:
Right, either. So there should be an OR, right? Otherwise if `errSeccomp` is non-nil and `errSelinux` is nil, the condition doesn't match and we continue.

Reply:
Yes, that's why I have separate check for  `errSeccomp` and `errSelinux` after that

Reply:
Oh I get it now, but in case only one of the lists has an error, we don't reconcile again and just ignore the error. Is that OK?

Reply:
ahh I see, I guess I should use or here instead.